stable/hive/templates/v3io-config-configmap.yaml

```diff
@@ -0,0 +1,153 @@
+    cp {{ .Values.global.v3io.configMountPath }}/hive-site.xml $V3IO_HIVE_CONFIG/
+
+    # Not working with 1.9.x, please uncomment for 2.0 once it ready.
+    #cp {{ .Values.global.v3io.configMountPath }}/core-site.xml $V3IO_HIVE_CONFIG/
+
+    # Initialize Hive metastore database
+    # Check if schema does not exists or invalid, then try to initialize Hive metastore database schema
+    set +e
+    $HIVE_HOME/bin/schematool -dbType $HIVE_DB_TYPE -validate
+    RC=$?
+    set -e
+    if [ $RC != 0 ]; then
+      echo "Hive schema didn't pass validation. RC=$RC"
+      echo "Initializing Hive metastore schema..."
+      $HIVE_HOME/bin/schematool -initSchema -dbType $HIVE_DB_TYPE
+      RC=$?
+      if [ $RC != 0 ]; then
+        echo "Failed to initialize Hive metastore database schema. RC=$RC"
+        exit $RC
+      else
+        echo "Hive metastore schema initialized successfully."
+      fi
+    else
+      echo "Hive metastore schema already exists. Details:"
+      $HIVE_HOME/bin/schematool -dbType $HIVE_DB_TYPE -info
+    fi
+
+    echo Starting hive metastore service....
+    $HIVE_HOME/bin/hive --service metastore
+
+  hive-site.xml: |
+    <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+    <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+    <configuration>
+{{ if eq .Values.metastore.database.type "postgresql" }}
+      <property>
+        <name>javax.jdo.option.ConnectionURL</name>
+        <value>jdbc:postgresql://{{ .Values.metastore.database.hostname }}:{{ .Values.metastore.database.port }}/{{ .Values.metastore.database.name }}</value>
+      </property>
+
+      <property>
+        <name>javax.jdo.option.ConnectionDriverName</name>
+        <value>org.postgresql.Driver</value>
+      </property>
+{{ else }}
+      <property>
+        <name>javax.jdo.option.ConnectionURL</name>
+        <value>jdbc:mysql://{{ .Values.metastore.database.hostname }}:{{ .Values.metastore.database.port }}/{{ .Values.metastore.database.name }}</value>
+      </property>
+
+      <property>
+        <name>javax.jdo.option.ConnectionDriverName</name>
+        <value>org.mariadb.jdbc.Driver</value>
+      </property>
+{{ end }}
+
+      <property>
+        <name>javax.jdo.option.ConnectionUserName</name>
+        <value>{{ .Values.metastore.database.username }}</value>
+      </property>
+
+      <property>
+        <name>javax.jdo.option.ConnectionPassword</name>
+        <value>{{ .Values.metastore.database.password }}</value>
+      </property>
+
+      <property>
+        <name>hive.metastore.uris</name>
+        <value>thrift://{{ template "hive.fullname" . }}:{{ .Values.metastore.containerPort }}</value>
+        <description>IP address (or fully-qualified domain name) and port of the metastore host</description>
+      </property>
+
+      <property>
+        <name>hive.metastore.schema.verification</name>
+        <value>true</value>
+      </property>
+    </configuration>
+
+  core-site.xml: |
```

Comment:
This exists in the container. Please use that version instead of the config map value.

Reply:
This file is little bit different. It uses different FS implementation with Hive.
```
      <property>
        <name>fs.v3io.impl</name>
        <value>io.iguaz.v3io.hcfs.V3IOHiveFileSystem</value>
      </property>
```